src/test/java/com/sorting/SelectionTest.java

```diff
@@ -0,0 +1,89 @@
+package com.sorting;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+
+import junit.framework.TestCase;
+
+public class SelectionTest extends TestCase {
+	
+	public void testSortWithString(){
+		
+		String[] actual = {"F","M","B","Z","Q","A"};
+		String[] expected = {"A","B","F","M","Q","Z"};
+		Selection.sort(actual);
+		//Selection.show(actual);
+		assertEquals(expected[2],actual[2]);
+		
+	}
+	
+public void testSortWithIntegers(){
+		
+		Integer[] actual = {9,4,3,6,5,2};
+		Integer[] expected = {2,3,4,5,6,9};
+		Selection.sort(actual);
+		//Selection.show(actual);
+		assertEquals(expected[4],actual[4]);
+		
+	}
+
+public void testSortwithValues(){
+	
+	Map<String,Integer> m1= new HashMap<String,Integer>();
```

Comment:
Remove Map, Set, Array etc. Simply creates a private inner class in this test class called "Transaction". Give some properties to it. Then in this method create an array of Transaction objects with properties filled. Implement comparable in transaction also. Then in this method call the sort (array of transactions) and compare with expected output. if you want to sort with different property then use comparator.